web/packages/teleterm/src/deepLinks.ts

```diff
@@ -28,7 +28,10 @@ export type DeepLinkParseResult =
   | ParseError<'unknown-protocol', { protocol: string }>
   | ParseError<'unsupported-uri'>;
```

Comment:
While you're at it, could you rename `unsupported-uri` to `unsupported-url`? I have no idea why I called it `uri` when everywhere else we use URL. If you decide to do so, please remember about changing the message here as well:

https://github.com/gravitational/teleport/blob/ceaeabc3ed6984da8082d90297bcda051e04dbc9/web/packages/teleterm/src/main.ts#L405